<!-- -->
<launch>

	<arg name="loop_rate" default="100.0"/>
	<arg name="acceleration_limit" default="3.0"/>
	<arg name="deceleration_limit" default="6.0"/>
	<arg name="max_curvature_rate" default="0.75"/>

	<node pkg="as" type="as_interface" name="as_interface" output="screen">
		<param name="loop_rate" value="$(arg loop_rate)" />
		<param name="acceleration_limit" value="$(arg acceleration_limit)" />
		<param name="deceleration_limit" value="$(arg deceleration_limit)" />
		<param name="max_curvature_rate" value="$(arg max_curvature_rate)" />
	</node>

</launch>
